doc/source/parameters/cfd/dynamic_flow_control.rst

```diff
@@ -0,0 +1,38 @@
+Dynamic flow control
+~~~~~~~~~~~~~~~~~~~~
+
+This subsection's purpose is to enable a dynamic flow control. It enables setting a volumetric flow on a specific boundary toward to normal of the wall. 
+It calculates a Beta coefficient at each time step and may be added to a source term. If the chosen wall boundary on which a volumetric flow is imposed is at inlet, 
+the volumetric flow rate targeted must be negative since outward normal vector is used. Thereby, if the chosen wall boundary is at outlet, 
+volumetric flow rate must be positive.
```

Comment:
```suggestion
This subsection's purpose is to enable a dynamic flow control. It enables setting a volumetric flow on a specific boundary toward to normal of the wall. 
It calculates a :math:`\beta` coefficient at each time step which may be added to a source term. If the chosen wall boundary on which a volumetric flow is imposed is an inlet, 
the volumetric flow rate targeted must be negative since outward normal vector is used. Thereby, if the chosen wall boundary is an outlet, 
volumetric flow rate must be positive.
```

Reply:
I really need to use the word "at". Now would it be better to use "at the" inlet/outlet?

Reply:
I agree for the beta part :)

Reply:
Maybe not at exclusively but something that means the same. It could also be on

Reply:
I think I would rewrite this part a little bit like this, taking into account Olivier's comments: 

```suggestion
The purpose of this subsection is to enable dynamic flow control. It is important when we want to target a volumetric flow on a specific boundary. To control the volumetric flow, the code calculates a :math:`\beta`  coefficient at each time step that is used to keep the volumetric flow value inside an acceptable threshold. If the chosen wall boundary on which a volumetric flow is imposed is an inlet, the volumetric flow rate targeted must be negative since the outward normal vector is used for the calculation. Thereby, if the chosen wall boundary is an outlet, volumetric flow rate must be positive.
```
Somehow, I feel that it sounds better to say that the wall is indeed an outlet or inlet instead of saying that it is located at the outlet or inlet.